### ISSUE TITLE
Plugin is not compatible. 

### ISSUE THREAD

**Issue by lance0821** [ncsoft/Unreal.js#70]
I have followed the instructions and installed version 4.11.2 of the engine, and placed the plugins folder in the project directory, but when starting my project I get an error stating some modules are not compatible with this version of the engine. The next popup points to V8 saying it was compiled against another version of the engine. Anyone else have this issue or an idea on how to resolve?

**Comment by nakosung** [comment]
Could you retry with latest build? I have uploaded a new build.

**Comment by lance0821** [comment]
Perfect Javascript console is now available and I tested 2+2 and it resulted in 4, so I assume all is well. Is there a guide or video to recompile the plugin for new engine versions? I've dabbled with UE4 since its release, but not too familiar to building the plugins. Now that I can use JS instead of Blueprints or C++ I would like to devote a little more time, and would like to minimize the headaches associated with incompatible versions and waiting on new builds. (Although you were super fast to respond and resolve issue)
